### DISCUSSION TITLE
Every update overwrites global editor setting

### DISCUSSION THREAD

**Discussion by vlad-datacat** [git-for-windows/git#3384] [Q&A] [answered]
- Every time a new version of git for windows released and I run the update it resets my default editor set via git --global.editor. I am using nano as my global editor, and it gets reset to VSCode.

 - What did you expect to occur after running these commands?

** My default editor to remain as my setting - nano. So if I type `git commit .` - it should as usual come up in Nano **

 - What actually happened instead?

** My default editor gets reset to VSCode. So if I type `git commit .` - it opens a VSCode new tab.  **

**Comment by rimrul** [comment, marked as answer]
In the installer uncheck "Only show new options" and choose `GNU nano` when it asks you what editor to use.

**Comment by dscho** [comment]
> 
> 
> In the installer uncheck "Only show new options" and choose `GNU nano` when it asks you what editor to use.

I wish the bug reporting template had not been deleted unceremoniously. If it had been filled out diligently, we would know whether the chosen editor really was VS Code.

In any case, I fully expect the recommended action to solve the issue, even if we haven't heard back in 3 days.
